operations/helm/tests/test-enterprise-values-generated/mimir-distributed/templates/alertmanager/alertmanager-config.yaml

```diff
@@ -0,0 +1,32 @@
+---
+# Source: mimir-distributed/templates/alertmanager/alertmanager-config.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-enterprise-values-mimir-alertmanager-fallback-config
+  labels:
+    app.kubernetes.io/name: mimir
+    app.kubernetes.io/instance: test-enterprise-values
+    app.kubernetes.io/component: alertmanager
+    app.kubernetes.io/part-of: memberlist
+    app.kubernetes.io/managed-by: Helm
+  annotations:
+    {} 
+  namespace: "citestns"
+data:
+  fallback-config.yaml: |
```

Comment:
this seems outdated. You should run `make build-helm-tests`.

I see there are conflicts as well, so better to first rebase onto grafana:main and then run that